.gitattributes

```diff
@@ -0,0 +1,2 @@
+**/prometheus_rules_out/*.yaml linguist-generated=true
+**/dashboards_out/*.json linguist-generated=true
```

Comment:
Missing new line

Reply:
These won't work unless Github has changed how Linguist work fundamentally. It doesn't support double use of `**`, we already attempted this last year